syft/pkg/cataloger/binary/default_classifiers.go

```diff
@@ -415,6 +417,94 @@ func DefaultClassifiers() []Classifier {
 			CPEs:    singleCPE("cpe:2.3:a:wp-cli:wp-cli:*:*:*:*:*:*:*:*"),
 		},
 	}
+
+	classifiers = append(classifiers, phpExtensionsClassifiers()...)
+
+	return classifiers
+}
+
+//nolint:funlen
+func phpExtensionsClassifiers() []Classifier {
```

Comment:
I don't quite understand the need for so many classifiers, can you help me understand what this is doing? (this should also probably be captured as a comment in the code)

Reply:
These are extensions for which the source exists in the PHP codebase but aren't compiled in the main PHP binary. They are optionally compiled when users need it. ex: https://github.com/nextcloud/docker/blob/master/28/fpm-alpine/Dockerfile#L41-L53